tests/integration/ledger/test_ledger.py

```diff
@@ -0,0 +1,116 @@
+# Require a connected Ledger ...
+
+import pytest
+
+from hexbytes import (
+    HexBytes,
+)
+
+from eth_account import (
+    Account,
+)
+from eth_account.messages import (
+    defunct_hash_message,
+)
+from eth_account.signers.ledger import (
+    LedgerAccount,
+)
+
+# Not a fixture because it does not support multiple LedgerAccount
+ledger = LedgerAccount()
+
+
+@pytest.fixture
+def acct(request):
+    return Account
```

Comment:
Looks like this could just be `Account` down below, and drop the `acct` fixture altogether.